doc/classes/CPUParticles2D.xml

```diff
@@ -297,7 +297,8 @@
 			Particles are drawn in the order emitted.
 		</constant>
 		<constant name="DRAW_ORDER_LIFETIME" value="1" enum="DrawOrder">
-			Particles are drawn in order of remaining lifetime.
+			Particles are drawn in order of remaining lifetime. In other words, the particle with the most lifetime is drawn on the front.
```

Comment:
```suggestion
			Particles are drawn in order of remaining lifetime. In other words, the particle with the highest lifetime is drawn at the front.
```
General grammar, "most" is used for countable things, like "most points", this is uncountable, can also be "greatest"

Same everywhere